PartyTime.lua

```diff
@@ -89,4 +86,10 @@ function Events:GROUP_ROSTER_UPDATE()
     T.AutoSetPartySymbols()
 end
 
+-- save assigned marker whenever one is set on a unit
+-- TODO should we save markers only for certain units (is a player, in party, etc?)
+function T.SetRaidTarget(unit, index)
+    --print("saving", MarkerFromIndex(index), "for", UnitName(unit))
+    GFW_PartyTime_SavedPresets[UnitName(unit)] = index
```

Comment:
Oops, missed one place for the short name.
```suggestion
    T.SavedPresets[UnitName(unit)] = index
```